### ISSUE TITLE
Crash when starting Windows

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4681]
**Reported by norrumar on 2014-12-10 10:27**
Sometimes I can't start Windows normally, as NVDA crashes and doesn't read at all.
If I press the Power button and try to start again, the computer starts Windows in Sequre mode, or a menu appears to choose how to start the system.
I don't see useful information in NVDA's log files.

**Comment by ehollig** [comment]
Has anyone experienced this in the past 3 years? As we are no longer able to contact the original poster of this issue, I am leaning toward a close here, unless the dmp file might provide some useful information in some underlining NVDA issue. Input from others would be appreciated.

**Comment by feerrenrut** [comment]
I had a quick look at this dump file, it didn't generate any leads. I'm going to close this issue. If anyone can provide more information, I would be happy to reopen.
